R/test.data.table.R

```diff
@@ -1,27 +1,34 @@
-test.data.table <- function(verbose=FALSE, pkg="pkg", silent=FALSE, with.other.packages=FALSE, benchmark=FALSE, Rraw=NULL) {
-  if (exists("test.data.table",.GlobalEnv,inherits=FALSE)) {
+test.data.table <- function(verbose=FALSE, pkg="pkg", silent=FALSE, with.other.packages=FALSE, benchmark=FALSE, script=NULL) {
```

Comment:
Renamed as any file extension will work. Also file doesn't have to be in `inst/tests`, so anyone can do own test script using our `test()` calls and run it `test.data.table(script="../myproj/my-dt-test-script.R")`. It works also from `cc("~/Desktop/my_dev_script.R")`.